src/internal/from-package/getImportMapFromPackageFiles.js

```diff
@@ -298,6 +298,16 @@ export const getImportMapFromPackageFiles = async ({
       })
       addMappingsForPackageAndImporter(mappingsFromPackageExports)
     } else {
+      // no "exports" field means any file can be imported
```

Comment:
This is the code "allowing" code to import any file from a package without `"exports"` field